build.gradle

```diff
@@ -43,6 +48,9 @@ allprojects {
     group 'com.palantir.human-readable-types'
     version gitVersion()
 
+    description = """This repository provides a collection of useful types that can be deserialized from human-readable strings.
```

Comment:
Do we need this to publish? Curious because if we do that's manual work to all our repos (or I guess we excavate out a placeholder description).

Reply:
Unfortunately yes, its a maven central requirement: https://central.sonatype.org/pages/requirements.html#project-name-description-and-url